Lib/test/test_future_stmt/test_future.py

```diff
@@ -52,45 +69,85 @@ def test_future_multiple_features(self):
         ):
             from test.test_future_stmt import test_future_multiple_features
 
-    def test_badfuture3(self):
-        with self.assertRaises(SyntaxError) as cm:
-            from test.test_future_stmt import badsyntax_future3
-        self.check_syntax_error(cm.exception, "badsyntax_future3", 3)
+    def test_unknown_future_flag(self):
+        code = """
+        '''Docstring'''
```

Comment:
Can you add one indentation level for inline code like this one, same in other tests?